contributing/getting-started-e2e.md

```diff
@@ -190,13 +190,23 @@ amphtml uses Node.js, the Yarn package manager and the Gulp build system to buil
     127.0.0.1 ads.localhost iframe.localhost
     ```
 
+* The AMP Project uses Gulp as our build system.   Gulp uses a configuration file ([gulpfile.js](https://github.com/ampproject/amphtml/blob/master/gulpfile.js)) to build amphtml (including the amphtml javascript) and to start up the Node.js server with the proper settings.  You don't really have to understand exactly what it is doing at this point--you just have to install it and use it.
+
+   You can install Gulp using Yarn:
+
+   ```
+   yarn global add gulp
```

Comment:
I thought we were going to recommend `gulp-cli`?

Reply:
Yes. See description:
> WIP. Not ready for review.

:)

Reply:
Done in the latest commit.